pkg/issuer/acme/dns/azuredns/azuredns.go

```diff
@@ -184,7 +185,7 @@ func (c *DNSProvider) createRecord(fqdn, value string, ttl int) error {
 		*rparams, nil)
 	if err != nil {
 		c.log.Error(err, "Error creating TXT:", z)
-		return err
+		return stabilizeError(err)
```

Comment:
This redacted error is accompanied by a log message which contains the complete error, for easier debugging.
While the case below does not.